app/assets/javascripts/test/fixtures/tasktracing_server_objects.js

```diff
@@ -0,0 +1,94 @@
+export const tracing = {
```

Comment:
Can you add type annotations to the exports so that we enforce that these fixtures are always in sync with our type definitions? I'm not sure how much flow typing is getting lost in the test files due to mocking & co.